private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/OwnerList/OwnerList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { connect } from "react-redux";
+import { OwnerType } from "../../../../../store/templateOwner/types";
+import { RootState } from "../../../../../store/rootReducer";
+import { ERROR_LOADING_IMAGE } from "../../../../../assets/strings";
+
+import { IFacepileProps, Facepile, OverflowButtonType, IFacepilePersona } from 'office-ui-fabric-react/lib/Facepile';
+
+interface Props {
+  owner?: OwnerType;
+  oids: string[];
+}
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    owner: state.templateOwner.owners,
+  };
+};
+const getPersonaProps = (persona: IFacepilePersona) => {	
+  return {	
+    hidePersonaDetails: true
+  }	
+};
+const facepileProps: IFacepileProps = {
```

Comment:
I'm wondering if we should go back to using Facepile to display the Author? I was having trouble with it re-rendering and that's why I stopped using it, but that might be fixed now.

I think we should be consistent in how we display the Author and people.